### PR TITLE
test: avoid rate limits in test_deployments when running in staging

### PR DESCRIPTION
**Pull request by tranchitella**
Changelog: none

Signed-off-by: Fabio Tranchitella <fabio.tranchitella@northern.tech>

### CODE REVIEW

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline

File: backend-tests/tests/test_deployments.py [hunk 1]
```diff
@@ -716,11 +716,19 @@ def do_test_regular_deployment(self, clean_mongo, user_token, devs):
             assert status_code == 200
             dev.artifact_name = deployment_req["artifact_name"]
 
+        # when running against staging, wait 5 seconds to avoid hitting
+        # the rate limits for the devices (one inventory update / 5 seconds)
+        isK8S() and time.sleep(5.0)
```
Comment (merlin-northern) [inline backend-tests/tests/test_deployments.py:721]: can we have `k8s_sleep_time=5.0` or something in those lines?

**Review by merlin-northern — APPROVED** [review]
pre-approv not to block

**Review by alfrunes — APPROVED** [review]

**Comment by tranchitella** [comment]
Force-pushed to use a constant as suggested.

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline

**Comment by tranchitella** [comment]
Failures are unrelated.

**Pull request merged**